client/multi-currency-setup/tasks/add-currencies-task/index.js

```diff
@@ -0,0 +1,339 @@
+					: activatedCount
+			) }
+			index={ 1 }
+		>
+			<CollapsibleBody>
+				<p className="wcpay-wizard-task__description-element is-muted-color">
+					{ ! isLoading &&
+						interpolateComponents( {
+							mixedString: __(
+								"Add currencies so international customers can shop and pay in their local currency. Your store's " +
+									'default currency is {{storeCurrencyText /}}.',
+								'woocommerce-payments'
+							),
+							components: {
+								storeCurrencyText: (
+									<strong>
+										{ StringRepresentationOfCurrency(
+											defaultCurrency
+										) }
+									</strong>
+								),
+							},
+						} ) }
+				</p>
+				{ 1 < enabledCurrenciesLength && (
+					<p className="wcpay-wizard-task__description-element is-muted-color">
+						{ interpolateComponents( {
+							mixedString: __(
+								"You've already added {{enabledCurrenciesText /}} to your store.",
+								'woocommerce-payments'
+							),
+							components: {
+								enabledCurrenciesText: (
+									<strong>
+										{ ConcatenateCurrencyStrings(
+											enabledCurrencyCodes,
+											defaultCurrencyCode,
+											availableCurrencies
+										) }
+									</strong>
+								),
+							},
+						} ) }
+					</p>
+				) }
+				<Card className="add-currencies-task__currency-selector-wrapper">
+					<CardBody>
+						<div className="add-currencies-task__search">
+							<Search
+								value={ searchText }
+								placeholder={ __(
+									'Search currencies',
+									'woocommerce-payments'
+								) }
+								onChange={ handleSearchChange }
+							/>
+						</div>
+						<div className={ 'add-currencies-task__separator' }>
+							&nbsp;
+						</div>
+						{ searchText ? (
+							/* translators: %1: filtered currencies count */
+							<h4>
+								{ sprintf(
+									__(
+										'Search results (%1$d currencies)',
+										'woocommerce-payments'
+									),
+									filteredCurrencyCodes.length
+								) }
+							</h4>
+						) : (
+							''
+						) }
+						<LoadableBlock numLines={ 30 } isLoading={ isLoading }>
+							<LoadableSettingsSection numLines={ 30 }>
+								<div className="add-currencies-task__content">
+									<EnabledCurrenciesModalCheckboxList>
+										{ ! searchText &&
+										recommendedCurrencyCodes.filter(
+											( code ) =>
+												! hiddenCurrencies.includes(
+													code
+												)
+										).length ? (
+											<>
+												<li>
+													<h4>
+														{ __(
+															'Recommended Currencies',
+															'woocommerce-payments'
+														) }
+													</h4>
+												</li>
+												{ availableCurrencyCodes.length
+													? recommendedCurrencyCodes.map(
+															displayCurrencyCheckbox
+													  )
+													: '' }
+												<li
+													className={
+														'add-currencies-task__separator'
+													}
+												>
+													&nbsp;
```

Comment:
Do we need the `&nbsp` here? Or it will be enough using an empty `li` like `<li ... />`

Reply:
I'll remove the `&nbsp;`s and try to make it work with CSS, but I'm not sure about it's browser compatibility. IIRC there were some cases that empty elements don't obey the padding and margin rules, maybe it was the old versions of IE (<= 9 or something).

Reply:
Done in 73cd066.